src/patternfly/components/Tabs/tabs.scss

```
@@ -531,6 +540,31 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
       content: revert;
     }
   }
+
+  &.pf-m-expanded {
+    --pf-c-tabs__item-toggle-icon--Color: var(--pf-global--Color--100);
+    --pf-c-tabs__item-toggle-icon--Rotate: var(--pf-c-tabs__item--m-expanded__toggle-icon--Rotate);
+  }
+
+  &:hover {
+    --pf-c-tabs__item-toggle-icon--Color: var(--pf-c-tabs__item--hover--Color);
+  }
+
+  &:active,
+  &.pf-m-active {
+    --pf-c-tabs__item-toggle-icon--Color: var(--pf-c-tabs__item--active--Color);
+  }
+
+  &:focus {
+    --pf-c-tabs__item-toggle-icon--Color: var(--pf-c-tabs__item--focus--Color);
+  }
+}
+
+.pf-c-tabs__item-toggle-icon {
+  font-size: small;
```

```suggestion
  font-size: var(--pf-c-tabs__item-toggle-icon--FontSize);
```

also you'll need to add `--pf-c-tabs__item-toggle-icon--FontSize: var(--pf-global--FontSize--sm)`